pkg/adapter/awsperformanceinsightssource/adapter.go

```diff
@@ -62,8 +61,7 @@ type adapter struct {
 	arn             arn.ARN
 	pollingInterval time.Duration
 	metricQueries   []*pi.MetricQuery
-	identifier      string
-	serviceType     string
+	resourceID      string
```

Comment:
Are we sure the RDS id the same as the PI id?
I did a test last week and they were different. The PI id was "db-longrandomstring" whereas the RDS id was the name I gave my insurance?